v2/log/log.go

```diff
@@ -121,6 +123,10 @@ func Base() Logger {
 	return baseLogger
 }
 
+func Nop() Logger {
+	return nopLogger
```

Comment:
Do we need the logger to be the same instance every time you ask for it? That might be true for Base() but here we might just,
```suggestion
	return logger{zap.NewNop().Sugar()}
```

Reply:
There is no need for the same instance for all Nop().
Chang to your suggestion and clean up is commited.